assets/css/style.css

```diff
@@ -0,0 +1,39 @@
+/* Root variables & imports */
+:root {
+    /* 1rem = 10px
+    This allows for greater control of calculating font sizes */
+    font-size: 62.5%;
+
+    /* Use of root variables to set typography, colors, utilites */
+
+    /* Typography */
+    /* Replace Arial/Tahoma with actual project fonts when ready */
+    --header-text: Arial, sans-serif;
+    --content-text: Tahoma, sans-serif;
+
+    /* Colors */
+    /* Replace and add to below with actual project color theme */
+    --p-color: #000000;
```

Comment:
why we using this color?

Reply:
@bellsmith15 Color changed to reflect a more spooky kind of deep purple. 🟪🟪🟪🟪🟪🟪